### ISSUE TITLE
find a way to remove invalid apk from the storage

### ISSUE THREAD

**Issue by vdelendik** [zebrunner/appium#313]
From time to time we see that apk is corrupted somehow and all sessions on this server/agent are broken.
let's try to capture and remove/mark apk as corrupted and required to be reuploaded...
Moreover, we might organize original copy everytime we download to be able to replace using non corrupted file.

```
org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/opt/android/platform-tools/adb -P 5037 -s 09271FDD4001R4 install -r /opt/appium-storage/.../111359.apk' exited with code 1'; Command output: adb: failed to install /opt/appium-storage/.../111359.apk: Failure [INSTALL_FAILED_INSUFFICIENT_STORAGE: Scanning Failed.: Package com....staging could not be assigned a valid UID] [[[--udid 0...4 --name Google_Pixel_5 --sessionId 144997ae-1be5-411e-a8bf-869b0b7c47eb]]] Host info: host: '7c5cd1ffff6d', ip: '172.18.0.62' Build info: version: '4.7.0', revision: '0a5b49d16f' System info: os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.0-142-generic', java.version: '17.0.9' 
```

**Comment by vdelendik** [comment]
Hi @SergeiZagriychuk 
How about implementing such steps to be able to identify corruption:
1. download artifact as usual
2. make a backup copy in the same folder
3. capture and print log size for original and backup everytime including the error/exception if sizes different

As only we confirm it is definitely a corruption of the file then we can arrange next steps to fix.

**Comment by vdelendik** [comment]
closing as won't fix because implementation starting from 2.0.x looks pretty stable
